### PR TITLE
Update Command.php

### PR DESCRIPTION
**Pull request by inSay**
The update post url structure is changed in the version 8

| Is bugfix?    | ✔️

### CODE REVIEW

**Comment by beowulfenator** [comment]
Thank you very much for the update. Could you please add Elasticsearch 8 to the tests? It's in https://github.com/yiisoft/yii2-elasticsearch/blob/master/.github/workflows/build.yml. Let's see if those tests pass. Thank you!

**Comment by tehmaestro** [comment]
Hi, I just hit this issue. Isn't this change also present in 7.x?

**Comment by beowulfenator** [comment]
I don't know, we're using AWS OpenSearch, whatever version it was forked from, and it seems to work.

**Comment by tehmaestro** [comment]
Hi, I can confirm that on 7.x  calling `/index/_doc/id/_update` only triggers a deprecation warning, but it does work. So I would say this pull request seems to do the right thing by checking for `dslVersion >= 8`.

**Comment by beowulfenator** [comment]
I understand, but can somebody please just add a single line to build.yml to add ES 8 to the testing matrix?

File: Command.php [hunk 1]
```diff
@@ -288,7 +288,9 @@ public function update($index, $type, $id, $data, $options = [])
             unset($options["detect_noop"]);
         }
 
-        if ($this->db->dslVersion >= 7) {
+        if ($this->db->dslVersion >= 8) {
+            return $this->db->post([$index, '_update', $id ], $options, Json::encode($body));
```
Comment (lubosdz) [inline Command.php:292]: Actually no need to distinguish for version 7 and 8 - new (fixed) syntax works for both well. 
In ES 7 this worked for BC compatability reasons also with ES 6 syntax.
https://www.elastic.co/guide/en/elasticsearch/reference/7.17/removal-of-types.html#_schedule_for_removal_of_mapping_types

**Comment by beowulfenator** [comment]
This is solved in #323.

**Comment by lubosdz** [comment]
Pls can be this released? It's been 3 months since the fix. The issue still appears after composer update & users report duplicity.

**Pull request closed without merging**